Modules/LearningSequence/classes/Player/class.ilLegacyKioskModeView.php

```diff
@@ -28,17 +28,14 @@ public function __construct(
 		$this->access = $access;
 	}
 
-	/**
-	 * @inheritDoc
-	 */
-	protected function getObjectClass(): string
+	protected function getObjectTitle(): string
```

Comment:
Do we still need this method?

Reply:
The method 'debugBuildAllControls' use this method.